dominikkawka/social-media-strike
# Ported original site to use semantic ui

# Description

Ported original POC to use semantic ui

## What

Imported semantic ui, created folder structure, imported jquery, used twitter api for feed.

## Why

To provide an alternative, and to show the use of semantic ui.

## Verification Steps

If you do not have a copy of the original repo locally run the following command 
```
git clone git@github.com:dominikkawka/social-media-strike.git
```
Now `cd` into the directory `social-media-strike`
Once in the directory you need to add my fork remote, this can be done by entering the following 
```
git remote add ciaranRoche git@github.com:ciaranRoche/social-media-strike.git
```
Now you have a remote to my fork of this repo, next you need to fetch my repo by using the following
```
git fetch ciaranRoche
```
Now you have access to all my branches associated with this repo, to checkout this particular branch of the PR you must enter the following
```
git checkout semantic-version
```
If you return to your local copy of this repo it should be updated to this branch, you can now open it in a browser and a code editor.
